node/index.js

```diff
@@ -60,23 +60,14 @@ const client = new plaid.Client({
 });
 
 const app = express();
-app.use(express.static('public'));
+// app.use(express.static('public'));
```

Comment:
make sure to clean up commented code here and in the dockerfile

Reply:
done